person.rb

```diff
@@ -0,0 +1,29 @@
+require './nameable'
```

Comment:
- [ ] [OPTIONAL] Though require can be used to both execute gems and external dependencies, the preferable method to load relative paths is `require_relative`. Therefore, I would encourage you to replace `require` with `require_relative` whenever you are loading relative paths and everywhere it occurs in your application in order to keep in line with best practices 😎